src/vs/workbench/browser/parts/quickopen/quickopen.ts

```diff
@@ -22,15 +22,15 @@ export class GlobalQuickOpenAction extends Action {
 	public static ID = 'workbench.action.quickOpen';
 	public static LABEL = nls.localize('quickOpen', "Go to File...");
 
-	constructor(id: string, label: string, @IQuickOpenService private quickOpenService: IQuickOpenService) {
+	constructor(id: string, label: string, @IQuickOpenService private quickOpenService: IQuickOpenService, @ICommandService private commandService: ICommandService) {
 		super(id, label);
 
 		this.order = 100; // Allow other actions to position before or after
 		this.class = 'quickopen';
 	}
 
-	public run(): TPromise<any> {
-		this.quickOpenService.show(null);
+	public run(prefix?: string, showOptions?: IShowOptions): TPromise<any> {
```

Comment:
I think we can remove `prefix` and `showOptions` from this action right? We are not using these arguments anywhere. The extension is using the command.